pyproject.toml

```diff
@@ -17,8 +17,8 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.7"
-nautobot = "^1.4.0"
+python = ">=3.8,<3.11"
```

Comment:
1.6 support 3.11 so I suspect 2.0 will as well... was thinking about this today can we remove this dependency definition and rely on that from Nautobot unless we have a real reason to limit python?

Reply:
I _think_ that you have to specify _a_ Python version constraint so that poetry works, but TBD.

Reply:
I actually can't:

```
❯ git diff
diff --git a/pyproject.toml b/pyproject.toml
index 22dc826..800834d 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -17,7 +17,7 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.11"
+python = ">=3.8"
 nautobot = "2.0.0-beta2"
 
 [tool.poetry.dev-dependencies]

nautobot-plugin-capacity-metrics on  2.0-migration [$!] is 📦 v2.0.0 via 🐍 v3.9.17 (nautobot-capacity-metrics-py3.9) 
❯ poetry lock --no-update
Resolving dependencies... (0.0s)

The current project's Python requirement (>=3.8) is not compatible with some of the required packages Python requirement:
  - nautobot requires Python >=3.8,<3.11, so it will not be satisfied for Python >=3.11

Because nautobot-capacity-metrics depends on nautobot (2.0.0b2) which requires Python >=3.8,<3.11, version solving failed.

  • Check your dependencies Python requirement: The Python requirement can be specified via the `python` or `markers` properties
    
    For nautobot, a possible solution would be to set the `python` property to ">=3.8,<3.11"

    https://python-poetry.org/docs/dependency-specification/#python-restricted-dependencies,
    https://python-poetry.org/docs/dependency-specification/#using-environment-markers
```